packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -191,7 +191,7 @@ contract EpochManager is
    */
   function startNextEpochProcess() external nonReentrant onlySystemAlreadyInitialized {
     require(isTimeForNextEpoch(), "Epoch is not ready to start");
-    require(!isOnEpochProcess(), "Epoch process is already started");
+    require(!isEpochProcessingStarted(), "Epoch process is already started");
```

Comment:
We can future proof it and require NotStarted instead (it also consumes a bit less gas)